tests/roots/test-ext-autosummary-recursive/package/package/package/package/module_importfail.py

```diff
@@ -0,0 +1,4 @@
+import sys
```

Comment:
Why is such deeper module needed? I think it is enough to test sub-package.

Reply:
Done